src/polynomial.jl

```diff
@@ -0,0 +1,49 @@
+# function Polynomial{C,E}(vec::AbstractVector, mat::AbstractMatrix) where {C <: Union{CxxWrap.CxxLong,VecOrMat_eltypes}, E <: Union{CxxWrap.CxxLong, Int64}}
+#     v = Vector{C}(vec)
```

Comment:
use `convert(Vector{C}, vec)`
also maybe change the names to `coefficients` and `exponents`?